packages/edit-site/src/components/dataviews/dataviews.js

```diff
@@ -1,211 +1,54 @@
-/**
- * External dependencies
- */
-import {
-	getCoreRowModel,
-	getFilteredRowModel,
-	getSortedRowModel,
-	getPaginationRowModel,
-	useReactTable,
-} from '@tanstack/react-table';
-
 /**
  * WordPress dependencies
  */
 import {
 	__experimentalVStack as VStack,
 	__experimentalHStack as HStack,
-	VisuallyHidden,
-	DropdownMenu,
-	MenuGroup,
-	MenuItem,
 } from '@wordpress/components';
-import { useMemo } from '@wordpress/element';
-import { __ } from '@wordpress/i18n';
 
 /**
  * Internal dependencies
  */
-import ListView from './list-view';
-import { Pagination } from './pagination';
+import ViewList from './view-list';
+import Pagination from './pagination';
 import ViewActions from './view-actions';
 import TextFilter from './text-filter';
-import { moreVertical } from '@wordpress/icons';
-
-const EMPTY_OBJECT = {};
+import { ViewGrid } from './view-grid';
 
 export default function DataViews( {
-	actions,
-	data,
-	fields,
 	view,
 	onChangeView,
-	isLoading,
+	fields,
+	actions,
+	data,
+	isLoading = false,
 	paginationInfo,
-	options: { pageCount },
```

Comment:
This prop duplicated an info we already had in paginationInfo so I just removed it.